torch/_dynamo/config.py

```diff
@@ -234,6 +234,28 @@
 # overlap without graph-breaks.
```

Comment:
should update the comments here to talk about the modes.  I think you should put the new strings up front, and document something like this

'no_optimization' (False): (description..)
'ddp_optimizer' (True): (description..)
'python_reducer': (experimental, description...)
...
and then add a 'Note: bool values are mapped for backwards compatibility as shown above'